evm/its.js

```diff
@@ -715,7 +689,8 @@ if (require.main === module) {
     program.addOption(new Option('--pauseStatus <pauseStatus>', 'pause status').choices(['true', 'false']));
     program.addOption(new Option('--rawSalt <rawSalt>', 'raw deployment salt').env('RAW_SALT'));
 
-    program.action((options) => {
+    program.action((command, options) => {
+        options.action = command;
```

Comment:
Avoid quick and dirty workarounds. Refactor it properly (see Sui scripts for reference)